js/modules/databindings/profileChart.js

```diff
@@ -1,12 +1,31 @@
 "use strict";
-define(['knockout', 'd3', 'd3-tip', 'lodash', 'd3-selection', 'webapi/MomentAPI', 'webapi/AuthAPI', 'appConfig', 'D3-Labeler/labeler'], function (ko, d3, d3tip, _, d3Selection, momentApi, authApi, config) {
+define([
```

Comment:
Why `js/modules/databindings/profileChart.js` and not `js/bindings/profileChart.js`?